scst-store/use-external-database.hbs.md

```diff
@@ -0,0 +1,42 @@
+# User External postgres database
+
+## <a id='prereqExtrenalDB'></a>Prerequisites
+
+* Setup your external postgres database. Once the database instance starts, retrieve the following information:
+
```

Comment:
Should we also mention they should setup their database to have TLS enabled? Or would this be assumed by the user by default?

Reply:
This is intentional